### PR TITLE
Adding node fields to MarkdownRemark Nodes

### PR DESCRIPTION
**Pull request by w8896699**
add labels into node, so md have validate label will show up in graphql

### CODE REVIEW

File: app-web/gatsby/onCreateNode.js [hunk 1]
```diff
@@ -236,17 +236,18 @@ module.exports = ({ node, actions, getNode, getNodes }) => {
       }
     });
 
-    let labels = [];
+    let labels = {};
     // assert the shape of labels in frontmatter
-    if (Object.prototype.hasOwnProperty(node.frontmatter, 'labels')) {
-      if (isArray(node.frontmatter.labels) && node.frontmatter.every(isString)) {
+    if (Object.prototype.hasOwnProperty.call(node.frontmatter, 'labels')) {
+      if (isObject(node.frontmatter.labels) && !isArray(node.frontmatter.labels)) {
```
Comment (w8896699) [inline app-web/gatsby/onCreateNode.js]: isObject(['aaa', 'bbb', 'ccc']) will return true... that's why I add isArrary here

Comment (patricksimonian) [reply app-web/gatsby/onCreateNode.js]: use isPlainObject instead.

File: app-web/gatsby/onCreateNode.js [hunk 2]
```diff
@@ -236,17 +236,18 @@ module.exports = ({ node, actions, getNode, getNodes }) => {
...
         labels = node.frontmatter.labels;
-      } else if (isString(node.frontmatter.labels)) {
-        // split by any commas and trim
-        labels = node.frontmatter.labels.split().map(text => text.trim());
+      } else {
+        // labels = node.frontmatter.labels.split().map(text => text.trim()); // --split by any commas and trim--
```
Comment (patricksimonian) [inline app-web/gatsby/onCreateNode.js]: remove commented line.

File: app-web/gatsby/onCreateNode.js [hunk 3]
```diff
@@ -236,17 +236,18 @@ module.exports = ({ node, actions, getNode, getNodes }) => {
...
+        labels = {};
```
Comment (patricksimonian) [inline app-web/gatsby/onCreateNode.js]: isn't labels already and empty object {}? this is redundant.

File: app-web/gatsby/createSchemaCustomization.js [hunk 4]
```diff
@@ -13,12 +13,16 @@ module.exports = ({ actions }) => {
       path: String
       id: String! 
     }
+    type frontmatterLableSet {
+      app: String
+      importance: String
```
Comment (patricksimonian) [inline app-web/gatsby/createSchemaCustomization.js]: I think we should leave importance out for now as there is no clear definition for it.

File: app-web/src/hoc/withResourceQuery.js [hunk 5]
```diff
@@ -56,6 +56,10 @@ const withResourceQuery = WrappedComponent => () => props => (
             node {
               fields {
                 title
+                labels {
```
Comment (patricksimonian) [inline app-web/src/hoc/withResourceQuery.js]: not using, so let's not query

**Pull request merged**